src/components/breadcrumbs/breadcrumbs.module.scss

```
@@ -4,28 +4,32 @@
-  display: inline-flex;
+  display: flex;
   align-items: center;
   height: 100%;
 
   .breadcrumb-item {
     position: relative;
-    display: inline-flex;
+    display: flex;
     align-items: center;
     max-width: 100%;
     @include font-scale();
```

Stylelint (17.3.0)

[error] Unexpected parentheses in argumentless mixin "font-scale" call (scss/at-mixin-argumentless-call-parentheses)

(scss/at-mixin-argumentless-call-parentheses)